code/modules/events/space_dragon.dm

```diff
@@ -0,0 +1,45 @@
+/datum/round_event_control/space_dragon
+	name = "Spawn Space Dragon"
+	typepath = /datum/round_event/ghost_role/space_dragon
+	max_occurrences = 1
+	weight = 8
+	earliest_start = 30 MINUTES
```

Comment:
this is to early, I would pitch at around 1 hour 10 or 1 hour 20

Reply:
This is the same weight as the pirates, and even they aren't too often of a spawn.  Also, most rounds are much shorter than 80 minutes, usually hovering around 50-60 minutes.  But, if that's what's truly needed...